website/skaffold.yaml

```diff
@@ -20,6 +20,7 @@ build:
           DEPLOY_ENVIRONMENT: staging
           IMPORT_SERVER_URL: https://staging.snackager.expo.io
           LEGACY_SNACK_SERVER_URL: https://staging.snack.expo.io
+          RUDDERSTACK_WRITE_KEY: 1weTbQObpRMmIsWrdwEITRabLA2
```

Comment:
Is this missing `RUDDERSTACK_DATA_PLANE_URL` ?

Reply:
yup, I added it in. In a recent update, I mistook `develop.env` for `base.env`